tools/expressionmerge.py

```diff
@@ -35,7 +36,7 @@
     base, ext = os.path.splitext(f)
     delimiter = ';' if ext == '.csv' else '\t'
 
-    with gzip.open(f, 'rb') as csvfile:
+    with io.TextIOWrapper(io.BufferedReader(gzip.open(f))) as csvfile:
```

Comment:
A quick test showed the buffers actually improve the performance, for almost 2-folds. Could not believe it, so i googled and found [this](http://aripollak.com/pythongzipbenchmarks/). It seems that there might be even better options (with popen and zcat), but I suggest we start by creating a function `gopen(file)` in `utils.py` that returns `io.TextIOWrapper(io.BufferedReader(gzip.open(file)))`. The we should replace all uses of gzip.open with our function. In the future we can improve the function with `zcat`.